generators/create-react-app/templates/deployment/kubernetes/docker/default.conf.ejs

```diff
@@ -0,0 +1,19 @@
+server {
```

Comment:
Missing noindex for staging env ?

Reply:
It's actually configure globally in the ingress configuration of our staging cluster.

And in the case where we deploy on another cluster it doesn't matter anyway since the staging should be behind an HTTP authentif..... but wait I forgot to add it :man_facepalming: